pkgs/development/ocaml-modules/ezxmlm/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, ocaml, findlib, dune, xmlm }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.2";
+  name = "ocaml${ocaml.version}-ezxmlm-${version}";
+
+  src = fetchFromGitHub {
+    owner = "avsm";
+    repo = "ezxmlm";
+    rev = "v${version}";
+    sha256 = "1dgr61f0hymywikn67inq908x5adrzl3fjx3v14l9k46x7kkacl9";
+  };
```

Comment:
Maybe the release artifact (https://github.com/avsm/ezxmlm/releases/download/v1.0.2/ezxmlm-1.0.2.tbz) is a better source.

Reply:
Tried using this and it [requires](https://github.com/avsm/ezxmlm/blob/master/pkg/pkg.ml#L3) [`topkg-jbuilder`](https://github.com/samoht/topkg-jbuilder), which I don't think is currently available in nixpkgs. Don't know if I'm missing something though :)